content/docs/rfcs/33/README.md

```diff
@@ -0,0 +1,104 @@
+---
+slug: 33
+title: 33/WAKU2-DISCV5
+name: WAKU v2 Discv5 Ambient Peer Discovery
+status: raw
+editor: Daniel Kaiser <danielkaiser@status.im>
+contributors:
+---
+
+# Abstract
+
+`33/WAKU2-DISCV5` specifies a modified version of [Ethereum's Node Discovery Protocol v5](https://github.com/ethereum/devp2p/blob/master/discv5/discv5.md) as a means for ambient node discovery.
+[10/WAKU2](/specs/10) uses the `33/WAKU2-DISCV5` ambient node discovery network for establishing a decentralized network of interconnected Waku2 nodes.
+The `33/WAKU2-DISCV5` discovery network is isolated from the Ethereum Discovery v5 network; this isolation greatly improves discovery efficiency.
+
+# Background and Rationale
+
+[11/WAKU2-RELAY](/specs/11) assumes the existence of a network of Waku2 nodes.
+For establishing and growing this network, new nodes trying to join the Waku2 network need a means of discovering nodes within the network.
+[10/WAKU2](/specs/10) supports the following discovery methods in order of increasing decentralization
+
+* hard coded bootstrap nodes
+* `DNS discovery`
+* `peer-exchange` protocol
+* `33/WAKU2-DISCV5` (specified in this document)
```

Comment:
Wondering if a list like this is not better suited to the "Discovery" section in RFC [10/WAKU2](/specs/10)? You could then just reference that section here to make the some point. For example, we don't really support "peer-exchange" yet, but will hopefully _soon_ ™. Since the list of supported discovery protocols change frequently, we'll have to update this in several places.

Reply:
agreed. Maybe we could leave it here as long as the draft is in raw state, and once it advances we could update 10/WAKU and substitue a reference for this list. Wdyt?